src/pages/api/events/on-pay.page.ts

```diff
@@ -0,0 +1,105 @@
+import { BigNumber } from '@ethersproject/bignumber'
+import {
+  ParticipantsDocument,
+  ParticipantsQuery,
+  QueryParticipantsArgs,
+} from 'generated/graphql'
+import { emailServerClient } from 'lib/api/postmark'
+import { sudoPublicDbClient } from 'lib/api/supabase'
+import client from 'lib/apollo/client'
```

Comment:
**suggestion** maybe `apolloClient`. might be good to export an `apolloClient` var from this file anyways instead of default export

Reply:
yeh agree, currently we use this everywhere. I think its better to change everrything all at once wdyt